src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -632,4 +635,29 @@ public static void createRunConfigTasks(final MinecraftExtension extension, fina
             RunConfigGenerator.createIDEGenRunsTasks(extension, prepareRuns, makeSrcDirs, additionalClientArgs);
         });
     }
+
+    public static String sha1Code(File file) throws IOException, NoSuchAlgorithmException {
```

Comment:
https://github.com/MinecraftForge/ForgeGradle/blob/FG_3.0/src/common/java/net/minecraftforge/gradle/common/util/HashFunction.java

Reply:
Yeah, I just found this function right now and I will push a new Commit very soon.